medusa/metadata/generic.py

```diff
@@ -970,7 +947,12 @@ def _retrieve_show_images_from_tmdb(self, show, img_type):
         # get TMDB configuration info
         tmdb.API_KEY = app.TMDB_API_KEY
         config = tmdb.Configuration()
-        response = config.info()
+        try:
+            response = config.info()
+        except RequestException as e:
```

Comment:
Because this is a direct call to the tmdb_api. And the indexerUnavailable is a Medusa indexer exception.
It could be a good change, to replace this with a call to our own tmdb indexer though.